src/Tribe/Linked_Posts.php

```diff
@@ -1057,18 +1057,27 @@ public function saved_linked_post_dropdown( $post_type, $current = null ) {
 
 		add_filter( 'tribe_events_return_all_linked_posts_if_none', '__return_true' );
 
-		$my_linked_posts = $this->get_linked_post_info(
-			$post_type,
-			[
-				'post_status' => [
-					'publish',
-					'draft',
-					'private',
-					'pending',
-				],
-				'author'      => $current_user->ID,
-			]
-		);
+		$available_post_status = [
+				'publish',
+				'draft',
+				'private',
+				'pending',
+		];
```

Comment:
Sorry about the nit pick. Some indention tweaks:
```suggestion
		$available_post_status = [
			'publish',
			'draft',
			'private',
			'pending',
		];
```

Reply:
Looks good otherwise